package.json

```diff
@@ -15,12 +15,12 @@
   "dependencies": {
     "@brown-ccv/behavioral-task-trials": "^1.0.0",
     "@fortawesome/fontawesome-free": "^5.9.0",
-    "bootstrap": "^4.3.1",
+    "bootstrap": "^5.2.0-beta1",
     "electron-log": "^3.0.9",
     "electron-squirrel-startup": "^1.0.0",
     "event-marker": "git+https://github.com/brown-ccv/event-marker.git",
     "execa": "^5.0.0",
-    "firebase": "^8.2.6",
+    "firebase": "^9.8.4",
```

Comment:
Firebase v9 imported cleaner than v8 ([known issue](https://github.com/firebase/firebase-js-sdk/issues/3711)), so I just went ahead and upgraded the dependency and used "compat" imports to work with our existing code.